src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagJobs.java

```diff
@@ -29,46 +37,106 @@
     private static final Logger LOG = LoggerFactory.getLogger(BugsnagJobs.class);
 
     /**
-     * Schedule each Bugsnag job based on the delay configuration parameters
+     * Schedule each Bugsnag job based on the delay configuration parameters.
      */
     public static void initialize() {
-        // First, send initial event data requests to seed the database with events on start-up. These requests will be
-        // processed by the BugsnagEventHandlingJob once it has been scheduled below.
-        String configId = "1";
-        BugsnagConfig bugsnagConfig = Persistence.bugsnagConfig.getOneFiltered(eq("configId", configId));
-        if (bugsnagConfig == null || bugsnagConfig.seedEventData) {
-            BugsnagEventRequestJob.triggerNewEventDataRequest(BugsnagDispatcher.EventDataRequestType.SEED);
-            if (bugsnagConfig == null) {
-                // First time seeding event data. Create config entry to record this so the process is not repeated on
-                // next start-up.
-                bugsnagConfig = new BugsnagConfig(configId, false);
-                Persistence.bugsnagConfig.create(bugsnagConfig);
-                LOG.info("Initiated seeding of event data requests");
-            } else {
-                // Event data seeding has already happened (hence record), but the value has been reset manually. This
-                // is useful if we want to reseed. E.g. Set seedEventData = false via Mongo DB and restart otp middleware.
-                bugsnagConfig.seedEventData = false;
-                Persistence.bugsnagConfig.replace(bugsnagConfig.id, bugsnagConfig);
-                LOG.info("Initiated reseeding of event data requests");
-            }
-        }
-
+        // Get latest sync tracker from database to determine how far back we need to sync.
+        Set<String> projectsRequested = seedDataIfNeeded();
+        Set<String> allProjects = MonitoredComponent.getComponentsByProjectId().keySet();
+        Set<String> projectsNotImmediatelyRequested = Sets.difference(allProjects, projectsRequested);
         LOG.info("Scheduling Bugsnag event data requests for every {} hours", BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_HOURS);
-        // Next, handle sending event data requests to Bugsnag on a daily basis.
+        // Schedule projects not included in the seed data requests to begin immediately.
         Scheduler.scheduleJob(
-            new BugsnagEventRequestJob(),
+            new BugsnagEventRequestJob(projectsNotImmediatelyRequested),
             0,
             BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_HOURS,
             TimeUnit.HOURS);
-
+        // Schedule projects already requested to offset by the job delay period.
+        Scheduler.scheduleJob(
+            new BugsnagEventRequestJob(projectsRequested),
+            BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_HOURS,
+            BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_HOURS,
+            TimeUnit.HOURS);
```

Comment:
Nit: On second pass, I suppose we could move this repeated call to scheduleJob into a private method that takes projectIds and an initial delay value.